dataplane/flags/flags.go

```diff
@@ -873,6 +873,13 @@ var (
 			EnvVar: "CB_WORKSPACE",
 		},
 	}
+	FlWithSdxOptional = BoolFlag{
+		RequiredFlag: OPTIONAL,
+		BoolFlag: cli.BoolFlag{
+			Name:  "withsdxready",
```

Comment:
We usually put dashes between words for flags. I would prefer something like "with-sdx-ready".